### PR TITLE
feat(TVNOW): update to RTL+

### PR DESCRIPTION
**Pull request by N0chteil**
## Description 
- Changed TVNOW to RTL+
- Fixed presence

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot_206
Screenshot_207
Screenshot_208
Screenshot_209
Screenshot_210

</details>

### CODE REVIEW

File: websites/R/RTL+/presence.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+	duration: number,
+	paused: boolean;
+
+presence.on("UpdateData", async () => {
+	const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			Math.floor(currentTime),
+			Math.floor(duration)
+		),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
```
Comment (theusaf) [inline websites/R/RTL+/presence.ts:22]: Imgur links are preferred over Discord asset names
```suggestion
			largeImageKey: "https://i.imgur.com/fMMsZfV.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/R/RTL+/presence.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	if (document.querySelector("#bitmovinplayer-video-player_container")) {
+		video = document.querySelector(
+			"#bitmovinplayer-video-player_container"
+		) as HTMLVideoElement;
+		({ currentTime, duration, paused } = video);
+
+		if (!isNaN(duration)) {
+			const footerEles = document.querySelectorAll("li.ng-star-inserted");
+
+			presenceData.smallImageKey = paused ? "pause" : "play";
```
Comment (theusaf) [inline websites/R/RTL+/presence.ts:35]: Ditto.

File: websites/R/RTL+/presence.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+					.trim();
+			}
+
+			presenceData.details = title;
+
+			if (duration <= 5) {
+				delete presenceData.startTimestamp;
+				delete presenceData.endTimestamp;
+
+				presenceData.smallImageKey = "pause";
```
Comment (theusaf) [inline websites/R/RTL+/presence.ts:79]: Ditto

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/R/RTL+/presence.ts [hunk 4]
```diff
@@ -0,0 +1,154 @@
+const presence = new Presence({
+		clientId: "1033504954763198545",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+	}),
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:7]: Better way of getting strings.
```suggestion
	}),
```

File: websites/R/RTL+/presence.ts [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:9]:
```suggestion
async function getStrings() {
	return presence.getStrings(
		{
		play: "general.playing",
		pause: "general.paused",
		}
	);
}

```

File: websites/R/RTL+/presence.ts [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+let title: string,
+	currentTime: number,
+	video: HTMLVideoElement,
+	duration: number,
+	paused: boolean;
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:14]:
```suggestion
	paused: boolean,
	strings: Awaited<ReturnType<typeof getStrings>>;
```

File: websites/R/RTL+/presence.ts [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+presence.on("UpdateData", async () => {
+	const [startTimestamp, endTimestamp] = presence.getTimestamps(
+			Math.floor(currentTime),
+			Math.floor(duration)
+		),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/fMMsZfV.png",
+		},
+		{ pathname, href } = document.location;
+
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:25]:
```suggestion

		strings = await getStrings();
		
```

File: websites/R/RTL+/presence.ts [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+		if (!isNaN(duration)) {
+			const footerEles = document.querySelectorAll("li.ng-star-inserted"),
+				cover: HTMLImageElement = document.querySelector("img.dvd-cover");
+
+			presenceData.smallImageKey = paused
+				? "https://i.imgur.com/C6mbMYz.png"
+				: "https://i.imgur.com/crCKEaC.png";
+			presenceData.smallImageText = paused
+				? (await strings).pause
+				: (await strings).play;
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:41]:
```suggestion
				? strings.pause
				: strings.play
```

File: websites/R/RTL+/presence.ts [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+			if (cover) presenceData.largeImageKey = cover.src;
+
+			presenceData.details = title;
+
+			if (duration <= 5) {
+				delete presenceData.startTimestamp;
+				delete presenceData.endTimestamp;
+
+				presenceData.smallImageKey = "https://i.imgur.com/C6mbMYz.png";
+				presenceData.smallImageText = (await strings).pause;
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:85]:
```suggestion
				presenceData.smallImageText = strings.pause
```

File: websites/R/RTL+/presence.ts [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/filme")) {
+		presenceData.details = "Browsing movies";
+		presenceData.startTimestamp = browsingTimestamp;
+	} else if (pathname.includes("/specials/")) {
+		presenceData.details = "Viewing special:";
+		presenceData.state = document.title;
+		presenceData.startTimestamp = browsingTimestamp;
```
Comment (DarkVillager) [inline websites/R/RTL+/presence.ts:143]: Also remove all of these put it at the presenceData and just delete it when a video is playing

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/R/RTL+/presence.ts (hunk 5 above)
Comment (EncryptedDev) [reply websites/R/RTL+/presence.ts:14]: Doesn't seem like this was resolved

File: websites/R/RTL+/presence.ts [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+			largeImageKey: "https://i.imgur.com/fMMsZfV.png",
+		},
+		{ pathname, href } = document.location;
+
+	if (!strings) strings = await getStrings();
+
+	if (document.querySelector("#bitmovinplayer-video-player_container")) {
+		video = document.querySelector(
+			"#bitmovinplayer-video-player_container"
+		) as HTMLVideoElement;
```
Comment (EncryptedDev) [inline websites/R/RTL+/presence.ts:34]:
```suggestion
		video = document.querySelector<HTMLVideoElement>(
			"#bitmovinplayer-video-player_container"
		);
```

File: websites/R/RTL+/presence.ts [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+
+			if (cover) presenceData.largeImageKey = cover.src;
+
+			presenceData.details = vidTitle;
+
+			if (video.duration <= 5) {
+				delete presenceData.startTimestamp;
+				delete presenceData.endTimestamp;
+
+				presenceData.smallImageKey = "https://i.imgur.com/C6mbMYz.png";
```
Comment (EncryptedDev) [inline websites/R/RTL+/presence.ts:86]: Used multiple times so you can use an `Assets` enum, shown here
https://github.com/PreMiD/Presences/blob/01054ec857bc5052c45a0ee39a044ce21dd0bf85/websites/M/MangaFreak/presence.ts#L4-L8

File: websites/R/RTL+/presence.ts [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+				presenceData.smallImageText = strings.pause;
+			}
+		}
+	} else if (pathname === "/") presenceData.details = "Viewing main page";
+	else if (pathname.includes("/serien/")) {
+		presenceData.details = "Viewing series:";
```
Comment (EncryptedDev) [inline websites/R/RTL+/presence.ts:92]: Remove `:` in all strings which include this, we don't like this punctuation in a presence as it doesn't make much sense

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/R/RTL+/presence.ts (hunk 5 above)
Comment (N0chteil) [reply websites/R/RTL+/presence.ts:14]: @EncryptedDev  https://github.com/PreMiD/Presences/blob/f4c5dea13b2af0aab930e3fbde123666ae9e333f/websites/R/RTL%2B/presence.ts#L4-L12

**Review by Bas950 — APPROVED** [review]

**Pull request merged**